.gitignore

```diff
@@ -1,4 +1,3 @@
-/vendor
```

Comment:
Why are you dropping vendor from `.gitignore`?

Reply:
Because I've upgraded to go modules. (The main perk is that it's super flexible to build from anywhere, you don't need to worry about `go path` or installing `dep`.)